.github/workflows/cherry-pick-next-to-master.yml

```diff
@@ -21,8 +21,8 @@ jobs:
         uses: carloscastrojumo/github-cherry-pick-action@v1.0.9
         with:
           branch: master
-          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
https://github.com/marketplace/actions/github-cherry-pick-action#action-inputs it says that the `GITHUB_TOKEN ` env token will be used by default.
We probably don't need any explicit definition.